lib/core/config.js

```diff
@@ -322,6 +322,10 @@ Config.prototype.loadWebServerConfigFile = function() {
 
   let webServerConfig = this._mergeConfig(configFilePath, configObject, false);
 
+  if (configFilePath === false) {
+    this.webServerConfig = {enabled: false};
+    return;
+  }
```

Comment:
could be: `if (!configFilePath)`

Reply:
Actually, I think it can be `undefined`, it would then use the defaults. But `false` is when specified by the user.